assets/src/scss/layout/_footer-social-media.scss

```diff
@@ -13,6 +13,7 @@
       margin-bottom: 24px;
       display: block;
       line-height: 1;
+      white-space: nowrap;
```

Comment:
I added this line because otherwise when there are less than 3 social media icons the text `Follow us` splits into 2 lines which looks ugly 😬